cmake/options.cmake

```diff
@@ -233,6 +233,11 @@ macro(drake_setup_options)
     "Google command-line flags processing library")
 
   drake_system_dependency(
```

Comment:
I think I would prefer us to always use the Drake version, but I will defer to whatever everyone else decides.